openquake/hazardlib/tests/shakemap/shakemapconverter_test.py

```diff
@@ -104,6 +110,22 @@ def test_shapefile(self):
         aae(test_data['val']['PGA'], data['val']['PGA'])
         aae(test_data['bbox']['minx'], data['bbox']['minx'])
 
+    def test_usgs_station_conversion(self):
+        # Check USGS to OQ conversion station data is working correctly
+        s_id = "usp000gvtu" # Emilia Romagna 2012 EQ
+        url = SHAKEMAP_URL.format(s_id)
+        text = urlopen(url).read()
```

Comment:
Tests downloading shakemaps and stations are FORBIDDEN here since they are slow. The right way is to use static files, manually downloaded from the USGS sites and possibly reduced in size.